### PR TITLE
Add test for make uninstall target

### PR DESCRIPTION
**Pull request by ldorau**
Requires:
- [x] #108 
- [x] #110

### CODE REVIEW

File: test/test_make_uninstall.sh [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+MATCH_UNINSTALLED_FILES="${WORKSPACE}/test/test_make_uninstall.txt"
+UNINSTALLED_FILES="${BUILD_DIR}/uninstall-files.txt"
+
+cd ${BUILD_DIR}
+make uninstall
+
+cd ${INSTALL_DIR}
+find | sort > ${UNINSTALLED_FILES}
+
+# check if 'make install' installed all and only required files
```
Comment (bratpiorka) [inline test/test_make_uninstall.sh:52]: please update the comment - 'make uninstall'? Also could you add a comment how does this work? Does this script checks if given directories are empty?

File: test/test_make_uninstall.txt [hunk 2]
```diff
@@ -0,0 +1,11 @@
+.
```
Comment (bratpiorka) [inline test/test_make_uninstall.txt:1]: how does this work? why do we add '.' entry here?

Comment (ldorau) [reply test/test_make_uninstall.txt:1]: This file is an output of the `find | sort` command. `.` stands for the current directory.

File: test/test_make_uninstall.sh (hunk 1 above)
Comment (ldorau) [reply test/test_make_uninstall.sh:52]: Done.
Yes, The `find` command prints out all types of general files (so also directories) found in the given directory.
After `make uninstall` there should be only empty directories left.
Done.

**Review by igchor — APPROVED** [review]

**Pull request merged**